bashplotlib/scatterplot.py

```diff
@@ -28,16 +28,25 @@ def get_scale(series, is_y=False, steps=20):
     return scaled_series
 
 
-def _plot_scatter(xs, ys, size, pch, colour, title, cs):
+def _plot_scatter(xs, ys, size, pch, colour, title, cs, xtitle, ytitle):
     plotted = set()
 
+    x_scale = get_scale(xs, False, size)
```

Comment:
I assigned these scales to variables because we were calling the `get_scale` function a lot and it was getting a bit messy.